syllabus.md

```diff
@@ -21,13 +21,21 @@ The course begins with an introduction to probability theory, with pencil-and-pa
 
 
 
-## Required Textbooks 
+## Textbooks 
+
+### Required Textbooks
 - The main text for the class is *Foundations of Agnostic Statistics*, written by Peter M. Aronow and Benjamin T. Miller. A physical copy of the book is available for approximately $30 ([Amazon Link](https://www.amazon.com/Foundations-Agnostic-Statistics-Peter-Aronow/dp/1316631141), [Cambridge University Press Link](https://www.cambridge.org/us/academic/subjects/politics-international-relations/research-methods-politics/foundations-agnostic-statistics?format=PB)). As well, for individuals who would like to have a digital copy of the book, it is available through the UC Library ([link](https://doi-org.libproxy.berkeley.edu/10.1017/9781316831762)).
 
 - There is a required course packet, which you may purchase at study.net.  This includes select chapters from other textbooks, namely *Probability and Statistics, 8th Edition* by Devore. 
 
+### Optional Textbooks
+The following textbooks are not required but are available for those who would like extra practice problems and/or additional exploration of course concepts. Both are available through the UC library
+
+- _Modern Mathematical Statistics with Applications_ by Devore and Berk ([library link](https://link-springer-com.libproxy.berkeley.edu/book/10.1007%2F978-1-4614-0391-3)) is very readable and has lots of worked examples. It's a good complement to Aronow and Miller's concise mathematical language.
+- _Elementary Probability and Applications_ by Durrett ([library link](https://ebookcentral-proquest-com.libproxy.berkeley.edu/lib/berkeley-ebooks/detail.action?docID=461127)) has a lot of examples for you to work through. It's mostly examples. Only one person can have access to this ebook at a time, so to avoid blockages consider buying a physical textbook for lik $30 on [amazon](https://www.amazon.com/Elementary-Probability-Applications-Rick-Durrett-dp-0521867568/dp/0521867568/ref=mt_other?_encoding=UTF8&me=&qid=). 
```

Comment:
Lik --> like

Reply:
I thought we found another way for students to access the Durrett book without the one-person-at-a-time problem?

Reply:
How... do... I... just... add... a... single... letter... from... this... review... interface..?

Reply:
```suggestion
- _Elementary Probability and Applications_ by Durrett ([library link](https://ebookcentral-proquest-com.libproxy.berkeley.edu/lib/berkeley-ebooks/detail.action?docID=461127)) has a lot of examples for you to work through. It's mostly examples. Only one person can have access to this ebook at a time, so to avoid blockages consider buying a physical textbook for about $30 on [amazon](https://www.amazon.com/Elementary-Probability-Applications-Rick-Durrett-dp-0521867568/dp/0521867568/ref=mt_other?_encoding=UTF8&me=&qid=). 
```